### PR TITLE
Prevent html injection in dashboard  - terms of service

### PR DESCRIPTION
**Pull request by nkiruka**
# Description
Replace html_safe with markdown to prevent html injection in terms of service

Terms of service - (finish-signup-page)
Screen Shot 2019-10-11 at 4 06 53 PM

- [jira](https://codedotorg.atlassian.net/browse/FND-723)

## Testing story
Test changes locally.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by Hamms — APPROVED** [review]
Is this the only place this string is used? If so, LGTM

**Comment by nkiruka** [comment]
@Hamms Good question.  If it is, I will update PR.

**Comment by nkiruka** [comment]
@Hamms I made additional changes to this PR, please review.

**Comment by nkiruka** [comment]
@Hamms 

Is html.safe still applicable even though there isn't html in the string (activerecord.attributes.user.name) below?
https://github.com/code-dot-org/code-dot-org/pull/31239/files#diff-b6c3a7da23968e5b9dbd2bf9d9da0ccdR35

<br /> seems to be more widely adopted to indicate a line break.  I checked redcarpet's repo on preferred syntax for line-breaks.  thoughts.
https://github.com/code-dot-org/code-dot-org/pull/31239/files#diff-b6c3a7da23968e5b9dbd2bf9d9da0ccdR37

File: dashboard/config/locales/en.yml [hunk 1]
```diff
@@ -54,9 +54,8 @@ en:
         gender: 'Gender'
         edit_header: 'Edit Account Details'
         finish_sign_up_header: 'Finish creating your account'
-        finish_sign_up_subheader: 'Fill out the following information to finish creating a Code.org account for <strong>%{email}</strong>.'
-        finish_sign_up_subheader_provider: 'Fill out the following information to finish creating a Code.org account with <strong>%{provider}</strong> for <strong>%{email}</strong>.'
-        reset_password_token: "The reset password link you have followed"
```
Comment (Hamms) [inline dashboard/config/locales/en.yml:59]: I don't believe this should be removed

**Pull request closed without merging**